src/remage.cc

```diff
@@ -85,6 +89,11 @@ int main(int argc, char** argv) {
       ->type_name("FILE");
   app.add_option("-o,--output-file", output, "Output file for detector hits")->type_name("FILE");
   app.add_flag("-w,--overwrite", overwrite_output, "Overwrite existing output files");
+  app.add_flag("-m,--merge", merge_out_files, "Merge output files");
```

Comment:
don't add wrapper args to the C++ program, add them to the wrapper via ArgParse and then use `parse_known_args()` to extract them and pass a correct argument list to remage-cpp

Reply:
@tdixon97 I already have working code for that when I tested this with Luigi. I will send you an example patch that should work